indico/modules/rb_new/client/js/modules/admin/SettingsPage.jsx

```diff
@@ -0,0 +1,220 @@
+                    <Placeholder.Line />
+                </Placeholder>
+            );
+        }
+
+        return (
+            <>
+                <Header as="h2">
+                    <Translate>General Settings</Translate>
+                </Header>
+                <FinalForm onSubmit={this.handleSubmit}
+                           initialValues={settings}
+                           initialValuesEqual={_.isEqual}
+                           subscription={{
+                               submitting: true,
+                               hasValidationErrors: true,
+                               pristine: true,
+                               submitSucceeded: true,
+                               dirtySinceLastSubmit: true,
+                           }}>
+                    {fprops => (
+                        <Form onSubmit={fprops.handleSubmit}
+                              success={fprops.submitSucceeded && !fprops.dirtySinceLastSubmit}>
+                            <Field name="tileserver_url" component={ReduxFormField} as="input"
+                                   format={formatters.trim} formatOnBlur
+                                   label={Translate.string('Tileserver URL')}
+                                   disabled={fprops.submitting}
+                                   validate={val => {
+                                       if (!val) {
+                                           return undefined;
+                                       } else if (!val.match(/https?:\/\/.+/)) {
+                                           return Translate.string('Please provide a valid URL');
+                                       } else {
+                                           const missing = ['{x}', '{y}', '{z}'].filter(x => !val.includes(x));
+                                           if (missing.length) {
+                                               return PluralTranslate.string(
+                                                   'Missing placeholder: {placeholders}',
+                                                   'Missing placeholders: {placeholders}',
+                                                   missing.length,
+                                                   {placeholders: missing.join(', ')}
+                                               );
+                                           }
+                                       }
+                                   }}>
+                                <p className="field-description">
+                                    <Translate>
+                                        If you want to use the map, specify the URL to a tileserver covering the
+                                        area in which your rooms are.
+                                    </Translate>
+                                </p>
+                            </Field>
+                            <Field name="booking_limit" component={ReduxFormField} as="input"
```

Comment:
I think we use camelCase notation for form fields in other forms. Why did you choose to use snake_case here?

Reply:
i had that initially, but then realized we camelize and snakify for no real reason. I totally understand doing it when we actually need to work with the keys a lot in JS, but otherwise i think having consistent names in frontend and backend is great e.g. to be able to search for them